scripts/run-with-strimzi.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+#
+# Copyright Kroxylicious Authors.
+#
+# Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+#
+
+set -eo pipefail
+
+if [[ -z $QUAY_ORG ]]; then
+  echo "Please set QUAY_ORG, exiting"
+  exit 1
+fi
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+. "${SCRIPT_DIR}/common.sh"
+cd "${SCRIPT_DIR}/.."
+
+if [[ "${QUAY_ORG}" != 'kroxylicious' ]]; then
+  echo "building and pushing image to quay.io"
+  PUSH_IMAGE=y "${SCRIPT_DIR}/deploy-image.sh"
+else
+  echo "QUAY_ORG is kroxylicious, not building/deploying image"
+fi
+set +e
+MINIKUBE_MEM=$(${MINIKUBE} config get memory)
+MINIKUBE_MEM_EXIT=$?
+set -e
+MINIKUBE_CONF='--memory=4096'
+if [ $MINIKUBE_MEM_EXIT -eq 0 ]; then
+  if [[ "$MINIKUBE_MEM" -lt 4096 ]]; then
+    echo "minikube memory is configured to below 4096 by user, overriding to 4096"
+  else
+    echo "minikube memory configured by user"
+    MINIKUBE_CONF=''
+  fi
+else
+  echo "no minikube memory configuration, defaulting to 4096M"
+fi
+INSTALL_DIR="sample-install"
+${MINIKUBE} start "${MINIKUBE_CONF}"
+${KUBECTL} create namespace kafka || true
+${KUBECTL} create -f 'https://strimzi.io/install/latest?namespace=kafka' -n kafka
```

Comment:
What would you think to:

```suggestion
${KUBECTL} create -f 'https://strimzi.io/install/latest?namespace=kafka' -n kafka --dry-run=client -o yaml | ${KUBECTL} apply -f -
```

Source: 
https://stackoverflow.com/questions/47369351/kubectl-apply-vs-kubectl-create

Reply:
I've changed it to an apply so running the script succeeds even if everything has been deployed before. 

We will need some more work in a subsequent PR to make the `update` case useful. Pushing out a new snapshot image to quay currently wouldn't update the pod, same with updating the config map, you still have to manually delete the kroxy pod to pick up changes.